conf/conf.go

```diff
@@ -87,8 +87,7 @@ func defaultConfig() config {
 		blockTxLimit: 1 << 16,
 	}
 
-	switch sys.VersionMeta {
-	case "testnet":
+	if sys.VersionMeta == "testnet" {
```

Comment:
This is better as a `switch` because we'll soon be adding "mainnet" and "choasnet";  We can add them in there now just with the same changes for "chaosnet".

On the other hand, we should consolidate all the places that these parameters are changed so maybe we should just have a different PR for that and not delay this :-)

Reply:
as soon as more values will be added, switch will be required, so its fine